src/forms/portfolio-share-form.schema.ts

```diff
@@ -1,7 +1,14 @@
+import Field from '@data-driven-forms/react-form-renderer/dist/cjs/field';
+import { SelectOptions } from '../types/common-types';
+import { Schema } from '@data-driven-forms/react-form-renderer';
```

Comment:
cjs import - I think you should be consistent, otherwise I hope that TS removes types imports from the final code. :smile: 

The same applies for other files in this PR.

Reply:
VS code auto imports :+1: